<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatForm</class>
 <widget class="QWidget" name="ChatForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>81</y>
     <width>671</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>380</y>
     <width>671</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="closeCurrentButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wayne</string>
   </property>
  </widget>
  <widget class="QLabel" name="signatureLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>861</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendFileButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="friendInfoBox">
   <property name="geometry">
    <rect>
     <x>679</x>
     <y>79</y>
     <width>191</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>好友信息</string>
   </property>
   <widget class="QLabel" name="numLabel">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>账号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="turenameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="sexLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>性别：</string>
    </property>
   </widget>
   <widget class="QLabel" name="ageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>年龄</string>
    </property>
   </widget>
   <widget class="QLabel" name="schoolLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>学校：</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>679</x>
     <y>369</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="fileTranLayout"/>
  </widget>
  <widget class="QPushButton" name="minButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>0</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeAllButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>0</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>472</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
